src/main/java/dk/kb/util/xml/LRUCache.java

```diff
@@ -0,0 +1,29 @@
+package dk.kb.util.xml;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Simple LRUcache based on the LinkedHashMap
+ *
+ * @see LinkedHashMap
+ */
+public class LRUCache<K, V> extends LinkedHashMap<K, V> {
```

Comment:
This does not look like a cache with [Least Recently Used](https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU) eviction. It looks like it is [First in, first out](https://en.wikipedia.org/wiki/FIFO_(computing_and_electronics)). In order for this to be LRU, elements should be re-inserted (to update their position in the `LinkedHashMap`) when they are retrieved from the cache.

Reply:
It should be controlled by accessOrder. Now accessOrder is hardcoded to true, so this WILL be a LRUCache

Reply:
My bad, got it now!